extensions/functions_comparison.yaml

```diff
@@ -214,3 +214,26 @@ scalar_functions:
         variadic:
           min: 2
         return: any1
+  -
+    name: "least"
```

Comment:
These functions (least and greatest) should follow the signature of coalesce, i.e., variadic.

Reply:
added. thanks!